ipfx/bin/run_feature_extraction.py

```diff
@@ -8,15 +8,15 @@
 from ipfx._schemas import FeatureExtractionParameters
 from ipfx.data_set_utils import create_data_set
 import ipfx.sweep_props as sp
-import ipfx.bin.make_stimulus_ontology as mso
 import allensdk.core.json_utilities as ju
 from allensdk.core.nwb_data_set import NwbDataSet
+import ipfx.nwb_writer as writer
 
 import ipfx.plot_qc_figures as plotqc
 import ipfx.logging_utils as lu
 
 
-def embed_spike_times(input_nwb_file, output_nwb_file, sweep_features):
+def embed_spike_times_old(input_nwb_file, output_nwb_file, sweep_features):
```

Comment:
If this is going to be `old` maybe might be good idea to add a deprecation warning? I'm guessing there's a reason it can't just be removed?

Reply:
good catch, I just need to remove the `old` one
Could you please comment on test class implementation?